packages/dialtone-css/README.md

```diff
@@ -1,14 +1,14 @@
-# Dialtone
+# Dialtone CSS
 
-This is the home for Dialtone, Dialpad's design system. It includes the resources needed to create consistent,
+This is the home for Dialtone CSS, Dialpad's design system. It includes the resources needed to create consistent,
 predictable interfaces that conform to Dialpad’s design principles, language, styles, and best practices.
```

Comment:
Can probably just remove this entire thing it is talking too much about Dialtone in general. Say something like

"This is the library for Dialtone's CSS. Any classes or styles used within Dialtone should be stored here and documented on our site in apps/dialtone-documentation."